vendors/espressif/boards/esp32/CMakeLists.txt

```diff
@@ -324,6 +325,34 @@ target_link_libraries(
 
 endif()
 
+# Common I/O
+afr_mcu_port(common_io)
+target_sources(
+    AFR::common_io::mcu_port
+    INTERFACE
+        "${afr_ports_dir}/common_io/iot_i2c.c"
+        "${afr_ports_dir}/common_io/iot_spi.c"
+        "${afr_ports_dir}/common_io/iot_uart.c"
+         $<${AFR_IS_TESTING}:${afr_ports_dir}/common_io/iot_test_common_io_internal.c>
+)
+target_include_directories(
+    AFR::common_io::mcu_port
+    INTERFACE
+        "${afr_ports_dir}/common_io/include"
+        "${AFR_ROOT_DIR}/libraries/abstractions/common_io/test"
+)
+
+
+target_compile_options(
```

Comment:
Why did we add more compile options?

Reply:
There were a bunch of compile errors when common-io is enabled. These compile options fix those errors.